TheMovieProject/Model/APIClient.swift

```diff
@@ -63,6 +63,16 @@ class APIClient {
                     completion(response.result)
             }
         }
+    static func loadGenre(api_key: String,
```

Comment:
APIClient içerisindeki fonksiyonların hepsi AF.request kullanıyor. Birincisi, bu bizim üçüncü parti framework'lere direkt bir dependency ile bağlanmamız demek. Bu sebepten bir kısa bir uzun önerim var:
1- Generic bir fonksiyon yazıp request, return type, decoder(optional, default parameter olarak da JSONDecoder) ve completion alacak. AF.Request kısmını oraya taşıyacaksın, böylece başka bir network kütüphanesine geçersen Alamofire'dan kurtulmak daha hızlı olacak.
2- İlk maddedeki generic fonksiyonu bir generic class haline getirip protocol tanımlayabilirsin. Alamofire'a dependent olmamamız açısından önemli bir adım olur. İleride istediğimiz kütüphane ile çalışabiliriz. Bunu bir session ile konuşalım. 

Bu yorum review için geçerli değil, PR'da bulunsun istedim